gw_eccentricity/gw_eccentricity.py

```diff
@@ -63,29 +63,52 @@ def measure_eccentricity(tref_in=None,
                          method="Amplitude",
                          dataDict=None,
                          num_orbits_to_exclude_before_merger=2,
+                         precessing=False,
                          extra_kwargs=None):
     """Measure eccentricity and mean anomaly from a gravitational waveform.
 
-    Eccentricity is measured using the GW frequency omega22(t) =
-    dphi22(t)/dt, where phi22(t) is the phase of the (2, 2) waveform
-    mode. We currently only allow time-domain, nonprecessing waveforms. We
-    evaluate omega22(t) at pericenter times, t_pericenters, and build a
-    spline interpolant omega22_pericenters(t) using those data
-    points. Similarly, we build omega22_apocenters(t) using omega22(t) at
-    the apocenter times, t_apocenters.
+    Eccentricity is measured using the GW frequency omega_gw(t) =
+    d(phase_gw)/dt, where phase_gw(t) is the phase of the (2, 2) waveform mode
+    for nonprecessing systems. For precessing systems, phase_gw is obtained
+    using an antisymmetric combination of (2, 2) and (2, -2) mode phases in the
+    coprecessing frame.
 
-    Using omega22_pericenters(t) and omega22_apocenters(t), we first
-    compute e_omega22(t), as described in Eq.(4) of arXiv:2302.11257. We
-    then use e_omega22(t) to compute the eccentricity egw(t) using Eq.(8)
-    of arXiv:2302.11257. Mean anomaly is defined using t_pericenters, as
-    described in Eq.(10) of arXiv:2302.11257.
+    Throughout this documentation, we will refer to omega_gw and amp_gw. For
+    nonprecessing systems, these quantities reduce to the corresponding values
+    obtained using the (2, 2) mode, i. e., for nonprecessing systems,
+
+    amp_gw = amp22, phase_gw = phase22 and omega_gw = omega22
+
+    On the other hand, for precessing systems, we use Eq.(48) and (49) of
+    arXiv:1701.00550 to define amp_gw and phase_gw. amp_gw (phase_gw) is
+    defined using a symmetric (antisymmetric) combination of
+    amplitude (phase) of (2, 2) and (2, -2) mode.
```

Comment:
Use [] instead of () to avoid mixing it up with (2,2) and (2,-2).
Say coprecessing.

Reply:
If you haven't already, please do a full read thru of the docs to see if things are clear.

Reply:
> Use [] instead of () to avoid mixing it up with (2,2) and (2,-2). Say coprecessing.

Done